File: src/public/billing/webhookApple.js
"use strict";

async function handleWebhookApple({ method, res }) {
  if (method !== "POST") return res.methodNotAllowed("Method not allowed");
  return res.notImplemented("TODO: Apple webhook verify signature + apply entitlements via internal route");
}

module.exports = { handleWebhookApple };

File: src_C/Public/Billing/WebhookApple.cs
using Amazon.Lambda.APIGatewayEvents;
using RecallSmith.Lambda.Common;

namespace RecallSmith.Lambda.Public.Billing;

public static class WebhookApple
{
  public static Task<APIGatewayProxyResponse> HandleWebhookApple(LambdaRequest req, Res res)
  {
    if (req.Method != "POST") return Task.FromResult(res.MethodNotAllowed("Method not allowed"));
    return Task.FromResult(res.NotImplemented("TODO: Apple webhook verify signature + apply entitlements via internal route"));
  }
}

